pcsx2-qt/MainWindow.cpp

```diff
@@ -1273,14 +1278,13 @@ void MainWindow::onInputRecNewActionTriggered()
 	}
 }
 
-#include "pcsx2/Recording/InputRecordingControls.h"
-
 void MainWindow::onInputRecPlayActionTriggered()
 {
 	const bool wasPaused = m_vm_paused;
-
 	if (!wasPaused)
-		g_InputRecordingControls.PauseImmediately();
+	{
+		VMManager::SetPaused(true);
```

Comment:
You shouldn't be calling VMManager functions on the UI thread. Always pass through g_emu_thread.